test/unit/mdc-textfield/mdc-text-field.test.js

```diff
@@ -88,6 +113,36 @@ test('#constructor instantiates a helper text on the element with id specified i
   document.body.removeChild(helperText);
 });
 
+test('#constructor instantiates an icon on the `.mdc-text-field__icon` element if present', () => {
+  const root = getFixture();
+  const component = new MDCTextField(root);
+  assert.instanceOf(component.icon_, MDCTextFieldIcon);
+});
+
+test('#constructor instantiates a label on the `.mdc-text-field__label` element if present', () => {
+  const root = getFixture();
+  const component = new MDCTextField(root);
+  assert.instanceOf(component.label_, MDCTextFieldLabel);
+});
+
+function setupTest(fixture) {
```

Comment:
I think you can do `fixture = getFixture()` for the parameter here instead of `fixture || getFixture()` for root below.